superdesk/validator.py

```diff
@@ -56,59 +98,61 @@ def _validate_type_email(self, field, value):
         # given that admins are usually create users, not users by themself,
         # probably just check for @ is enough
         # https://davidcel.is/posts/stop-validating-email-addresses-with-regex/
-        if not re.match('.+@.+', value, re.IGNORECASE):
-            self._error(field, ERROR_PATTERN)
+        if re.match('.+@.+', value, re.IGNORECASE):
+            return True
 
-    def _validate_type_file(self, field, value):
+    def _validate_type_file(self, value):
         """Enables validation for `file` schema attribute."""
-        if not isinstance(value, FileStorage):
-            self._error(field, ERROR_PATTERN)
+        if isinstance(value, FileStorage):
+            return True
 
     def _validate_multiple_emails(self, multiple, field, value):
         """
+        {'type': 'boolean'}
+
         Validates comma separated list of emails.
 
         :param field: field name.
         :param value: field value.
         """
         if multiple:
             emails = value.split(',')
-            for email in emails:
-                self._validate_type_email(field, email)
+            if not all([self._validate_type_email(email) for email in emails]):
+                self._error(field, ERROR_PATTERN)
 
     def _validate_unique(self, unique, field, value):
-        """Validate unique with custom error msg."""
-
+        """
```

Comment:
I got a lot of warnings after the develop server was started.

Almost all these warnings are like:
```
UserWarning: No validation schema is defined for the arguments of rule 'empty'
  "'%s'" % method_name.split('_', 2)[-1]
```

I think it can be solved by adding exact this price of docstring:

```
The rule's arguments are validated against this schema:
{'type': 'boolean'}
```